Paladin.h

```diff
@@ -1,7 +1,14 @@
 #pragma once
-
-struct Paladin : Character
+#include "Character.h"
+#include <iostream>
```

Comment:
you are not using anything in `iostream` in this header.  don't include what you don't use.
- [ ] revise.